doc/releases/changelog-dev.md

```diff
@@ -427,6 +427,11 @@
 
 <h3>Deprecations</h3>
 
+
+* `qml.transforms.measurement_grouping` has been removed. Users should use `qml.transforms.hamiltonian_expand`
+  instead.
+  [(#3701)](https://github.com/PennyLaneAI/pennylane/pull/3701)
+
```

Comment:
Since it removes deprecated content, it should go under "breaking change" instead. The "deprecations" section is for things that are now deprecated.

Reply:
Thanks for letting me know :)